html/includes/graphs/application/rrdcached.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$rrd = $config['rrd_dir'].'/'.$device['hostname'].'/app-rrdcached-'.$app['app_id'].'.rrd';
```

Comment:
Please have a look at using rrd_name() in includes/rrdtool.inc.php instead of this.